pages/scholars/_id.vue

```diff
@@ -1,13 +1,300 @@
+                      src="https://assets.coingecko.com/coins/images/10366/large/SLP.png?1578640057"
+                      max-height="35"
+                      max-width="35"
+                  ></v-img>
+                  <div class="headline-1 mx-3">
+                    0
+                  </div>
+                </v-row>
+              </v-card-title>
+                <v-row justify="center">
+                  <v-card-title>Ronin SLP</v-card-title>
+                </v-row>
+            </v-card>
+        </v-col>
+        <v-col
+            cols="12"
+            md="2"
+            sm="2"
+        >
+            <v-card
+              min-height="70"
+              
+            >
+                <v-card-title>
+                  <v-row
+                    class="pt-5 px-5"
+                    align="center"
+                    justify="center"
+                  >
+                      <v-img
+                          src="https://assets.coingecko.com/coins/images/10366/large/SLP.png?1578640057"
+                          max-height="35"
+                          max-width="35"
+                      ></v-img>
+                    <div class="headline-1 mx-3">
+                      {{ dailySLP }} 
+                    </div>
+                  </v-row>
+                </v-card-title>
+                <v-row justify="center">
+                  <v-card-title>Daily Average</v-card-title>
+                </v-row>
+            </v-card>
+        </v-col>
+        <v-col
+            cols="12"
+            md="2"
+            sm="2"
+        >
+            <v-card
+              min-height="70"
+              
+            >
+                <v-card-title>
+                  <v-row
+                    class="pt-5 px-5"
+                    align="center"
+                    justify="center"
+                  >
+                      <v-img
+                          src="https://assets.coingecko.com/coins/images/10366/large/SLP.png?1578640057"
+                          max-height="35"
+                          max-width="35"
+                      ></v-img>
+                    <div class="headline-1 mx-3">
+                      {{ farmedToday }} 
+                    </div>
+                  </v-row>
+                </v-card-title>
+                <v-row justify="center">
+                  <v-card-title>Farmed Today</v-card-title>
+                </v-row>
+            </v-card>
+        </v-col>
+        <v-col
+            cols="12"
+            md="2"
+            sm="2"
+        >
+            <v-card
+              min-height="70"
+            >
+                <v-card-title>
+                  <v-row
+                    class="pt-5 px-5"
+                    align="center"
+                    justify="center"
+                  >
+                    <v-img
+                      class="rounded-circle"
+                      src="https://chimeratribune.com/wp-content/uploads/2020/11/axieareanaswords.png"
```

Comment:
Refactor and move this inside `state` from `store/index.js` file, move it in a property called `assets` and have a key name of `axieareanaswords`
```js
// index.js

export const state = () => ({
  // ...
  assets: {
   // ...
   axieareanaswords: 'https://chimeratribune.com/wp-content/uploads/2020/11/axieareanaswords.png'
  }
})
```
Then from the view/page that requires to call this asset, call it by `$store.state.assets?.axieareanaswords`